.gitignore

```diff
@@ -1,2 +1,46 @@
-node_modules
-dist
\ No newline at end of file
+### STANDARD GIT IGNORE FILE ###
+
+# DEPENDENCIES
+node_modules/
+/.pnp
+.pnp.js
+
+# TESTING
+/coverage
+*.lcov
+.nyc_output
+
+# BUILD
+build/
+public/build/
+dist/
+generated/
+
+# ENV FILES
+.env
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+# LOGS
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# MISC
+.idea
+.turbo/
+.cache/
+.next/
+.nuxt/
```

Comment:
remove nuxt here to keep it clean

Reply:
Its just a generic .gitignore that covers 90% of use cases. If it doesn't hurt anything I would rather keep it complete and easily extensible to other projects.